addon.xml

```diff
@@ -65,194 +65,6 @@ v0.15.0 (2019-08-20)
 -Fixed UpNext watched status from Kodi library
 -Fixed issue with library items that containing % in the path
 -Other minor improvements and fixes
```

Comment:
insted you can move the changelog in a changelog.md file
in a separated commit

Reply:
Ok, but as a result this will not be part of the released ZIP file. And therefore not be available on the Kodi system.

Reply:
so we only keep only the change of the last version in addon.xml

Reply:
Well, some add-ons do it like that, but we prefer to keep a few released in the add-on.xml, because most users won't look at it until something works/behaves differently, which may not have been added in this release, but a release before. As long as it is inside the 1500 characters it is fine.

Reply:
Fixed.